### PR TITLE
Fixed: More improvements to sentry logging

### PR DESCRIPTION
**Pull request by ta264**
#### Database Migration
NO

#### Description
- Cleanse exception messages
- Don't add exception messages to fingerprints for all exception types to improve grouping
- Don't explicitly log exception messages - it happens automatically and breaks grouping

### CODE REVIEW

File: src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs [hunk 1]
```diff
@@ -210,7 +210,7 @@ private ValidationFailure TestConnection()
             }
             catch (DownloadClientAuthenticationException ex)
             {
-                _logger.Error(ex, ex.Message);
```
Comment (Qstick) [inline src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs]: Removing these causes the trace to log twice in console and local logs

**Comment by Qstick** [comment]
Does this also fix cases like this where the artist ID is different which causes sentry not to group the entries?

image

File: src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs (hunk 1 above)
Comment (ta264) [reply src/NzbDrone.Core/Download/Clients/Deluge/Deluge.cs]: Fixed in the updated version - I've replaced `ex.Message` with a generic string.

**Comment by ta264** [comment]
Yes, it should fix those cases too.

**Pull request merged**